Removed mesos as cloud provider from Kubernetes.

gyliu513 opened the pull request:
<!--  Thanks for sending a pull request!  Here are some tips for you:
1. If this is your first time, read our contributor guidelines https://git.k8s.io/community/contributors/devel/pull-requests.md#the-pr-submit-process and developer guide https://git.k8s.io/community/contributors/devel/development.md#development-guide
2. If you want *faster* PR reviews, read how: https://git.k8s.io/community/contributors/devel/pull-requests.md#best-practices-for-faster-reviews
3. Follow the instructions for writing a release note: https://git.k8s.io/community/contributors/devel/pull-requests.md#write-release-notes-if-needed
-->

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #47205

**Special notes for your reviewer**:

**Release note**:
<!--  Steps to write your release note:
1. Use the release-note-* labels to set the release note state (if you have access)
2. Enter your extended release note in the below block; leaving it blank means using the PR title as the release note. If no release note is required, just write `NONE`.
-->
```release-note
Move Mesos Cloud Provider out of Kubernetes Repo
```

gyliu513 committed:
Removed mesos as cloud provider from Kubernetes.

dims commented:
@gyliu513 Moving this code to an external cloud provider?

luxas commented:
Yay, moving these things out-of-monorepo is a great step for the project! 
Thanks @gyliu513 for making that possible!

I'd edit this to add a release note though :smile:

gyliu513 commented:
@dims Yes, there are some discussion at at xref ##34924 , but I'd like we first move this out of the repo of kubernetes as first step.

gyliu513 commented:
/release-note

gyliu513 commented:
@luxas done, thanks ;)

luxas commented:
@thockin woot, let's get this LGTM'd :)

k82cn commented:
sorry for the confusion by (#34924), kube-mesos still need this provider to get all hosts from Mesos until external cloud-provider is done.

luxas commented:
@k82cn Is there any tracking issue for moving mesos out?

k82cn commented:
@ luxas , #34924 is used to trace moving mesos out; let me update sub-tasks and dependencies.

k82cn commented:
@luxas , #34924 is used to trace moving mesos out; let me update sub-tasks and dependencies.

gyliu513 commented:
I think the kube-mesos project is dead and I belive no one is working on it, I think that it is time to retire it from Kubernetes. @luxas

luxas commented:
> I think the kube-mesos project is dead and I belive no one is working on it, I think that it is time to retire it from Kubernetes.

SGTM, the incubation project kube-mesos-framework hasn't been updated in ~9 months so...

k82cn commented:
> SGTM, the incubation project kube-mesos-framework hasn't been updated in ~9 months so...

I think China Unicom or [Didi](http://www.didichuxing.com/en/) maybe using that, and I'm not sure whether Mesosphere will include k8s in its DC/OS . Let me check whether anyone of them would like to invest resource for this project.

luxas commented:
@k82cn Anyway, that's not blocking this PR, or is it?

k82cn commented:
@luxas , let me check the status of "pluggable cloud providers" in 1.7; if it's ready, I think we can continue to merge this PR.

k82cn commented:
@luxas , as we slack yesterday, ccm is enough for kube-mesos; so I think we can merge this PR :).

k82cn commented:
/lgtm

luxas commented:
/lgtm

/assign @thockin 
for approval

cc @wlan0

mikedanese commented:
/approve

luxas commented:
/retest

> On 09 Jul 2017, at 11:03, k8s-ci-robot <notifications@github.com> wrote:
> 
> @gyliu513: The following tests failed, say /retest to rerun them all:
> 
> Test name	Commit	Details	Rerun command
> pull-kubernetes-unit	0ec9e3c	link	/test pull-kubernetes-unit
> pull-kubernetes-federation-e2e-gce	0ec9e3c	link	/test pull-kubernetes-federation-e2e-gce
> Full PR test history. Your PR dashboard. Please help us cut down on flakes by linking to an open issue when you hit one in your PR.
> 
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>

sttts commented:
Might be worth to run godep-save.sh again, to get rid of dependencies.

luxas commented:
Ok to merge as-is and follow up on that?

> On 09 Jul 2017, at 11:58, Dr. Stefan Schimanski <notifications@github.com> wrote:
> 
> Might be worth to run godep-save.sh again, to get rid of dependencies.
> 
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>

k82cn commented:
/retest

gyliu513 commented:
/retest

mikedanese commented:
We should do it now. Why would we wait?

luxas commented:
Just asked. Ok for me to do now.

> On 09 Jul 2017, at 14:37, Mike Danese <notifications@github.com> wrote:
> 
> We should do it now. Why would we wait?
> 
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>

gyliu513 commented:
/retest

luxas commented:
@gyliu513 Please remove the old mesos deps...

gyliu513 commented:
>@gyliu513 Please remove the old mesos deps...

@luxas Can you please show more detail for how to remove the old mesos deps? 

I was trying the following steps:

```
root@pc005:~/go/src/k8s.io/kubernetes# ./hack/godep-save.sh
The directory vendor/ or Godeps/ exists. Remove them before running godep-save.sh
```

Then I removed both vendor/ and Godeps/, and then run  ./hack/godep-save.sh, after this script finished, I found quite a lot of files are deleted.
```
modified:   Godeps/Godeps.json
	modified:   Godeps/LICENSES
	deleted:    vendor/BUILD
	deleted:    vendor/bitbucket.org/bertimus9/systemstat/BUILD
	deleted:    vendor/bitbucket.org/ww/goautoneg/BUILD
	deleted:    vendor/cloud.google.com/go/compute/metadata/BUILD
	deleted:    vendor/cloud.google.com/go/internal/BUILD
	deleted:    vendor/github.com/Azure/azure-sdk-for-go/arm/compute/BUILD
	deleted:    vendor/github.com/Azure/azure-sdk-for-go/arm/containerregistry/BUILD
	deleted:    vendor/github.com/Azure/azure-sdk-for-go/arm/network/BUILD
	deleted:    vendor/github.com/Azure/azure-sdk-for-go/arm/storage/BUILD
	deleted:    vendor/github.com/Azure/azure-sdk-for-go/storage/BUILD
	deleted:    vendor/github.com/Azure/go-ansiterm/BUILD
	deleted:    vendor/github.com/Azure/go-autorest/autorest/BUILD
	deleted:    vendor/github.com/Azure/go-autorest/autorest/azure/BUILD
	deleted:    vendor/github.com/Azure/go-autorest/autorest/date/BUILD
	deleted:    vendor/github.com/Azure/go-autorest/autorest/to/BUILD
	deleted:    vendor/github.com/Azure/go-autorest/autorest/validation/BUILD
	deleted:    vendor/github.com/MakeNowJust/heredoc/BUILD
	deleted:    vendor/github.com/Microsoft/go-winio/BUILD
	deleted:    vendor/github.com/NYTimes/gziphandler/BUILD
	deleted:    vendor/github.com/PuerkitoBio/purell/BUILD
	deleted:    vendor/github.com/PuerkitoBio/urlesc/BUILD
	deleted:    vendor/github.com/Sirupsen/logrus/BUILD
	deleted:    vendor/github.com/abbot/go-http-auth/BUILD
	deleted:    vendor/github.com/appc/spec/schema/BUILD
	deleted:    vendor/github.com/appc/spec/schema/common/BUILD
	deleted:    vendor/github.com/appc/spec/schema/types/BUILD
	deleted:    vendor/github.com/appc/spec/schema/types/resource/BUILD
	deleted:    vendor/github.com/armon/circbuf/BUILD
	deleted:    vendor/github.com/asaskevich/govalidator/BUILD
	deleted:    vendor/github.com/aws/aws-sdk-go/aws/BUILD
	deleted:    vendor/github.com/aws/aws-sdk-go/aws/awserr/BUILD
	deleted:    vendor/github.com/aws/aws-sdk-go/aws/awsutil/BUILD
	deleted:    vendor/github.com/aws/aws-sdk-go/aws/client/BUILD
	deleted:    vendor/github.com/aws/aws-sdk-go/aws/client/metadata/BUILD
	deleted:    vendor/github.com/aws/aws-sdk-go/aws/corehandlers/BUILD
	deleted:    vendor/github.com/aws/aws-sdk-go/aws/credentials/BUILD
	deleted:    vendor/github.com/aws/aws-sdk-go/aws/credentials/ec2rolecreds/BUILD
	deleted:    vendor/github.com/aws/aws-sdk-go/aws/credentials/endpointcreds/BUILD
	deleted:    vendor/github.com/aws/aws-sdk-go/aws/credentials/stscreds/BUILD
	deleted:    vendor/github.com/aws/aws-sdk-go/aws/defaults/BUILD
	deleted:    vendor/github.com/aws/aws-sdk-go/aws/ec2metadata/BUILD
	deleted:    vendor/github.com/aws/aws-sdk-go/aws/endpoints/BUILD
	deleted:    vendor/github.com/aws/aws-sdk-go/aws/request/BUILD
	deleted:    vendor/github.com/aws/aws-sdk-go/aws/session/BUILD
	deleted:    vendor/github.com/aws/aws-sdk-go/aws/signer/v4/BUILD
	deleted:    vendor/github.com/aws/aws-sdk-go/private/protocol/BUILD
	deleted:    vendor/github.com/aws/aws-sdk-go/private/protocol/ec2query/BUILD
	deleted:    vendor/github.com/aws/aws-sdk-go/private/protocol/json/jsonutil/BUILD
	deleted:    vendor/github.com/aws/aws-sdk-go/private/protocol/jsonrpc/BUILD
	deleted:    vendor/github.com/aws/aws-sdk-go/private/protocol/query/BUILD
	deleted:    vendor/github.com/aws/aws-sdk-go/private/protocol/query/queryutil/BUILD
	deleted:    vendor/github.com/aws/aws-sdk-go/private/protocol/rest/BUILD
	deleted:    vendor/github.com/aws/aws-sdk-go/private/protocol/restxml/BUILD
	deleted:    vendor/github.com/aws/aws-sdk-go/private/protocol/xml/xmlutil/BUILD
	deleted:    vendor/github.com/aws/aws-sdk-go/private/waiter/BUILD
	deleted:    vendor/github.com/aws/aws-sdk-go/service/autoscaling/BUILD
	deleted:    vendor/github.com/aws/aws-sdk-go/service/ec2/BUILD
	deleted:    vendor/github.com/aws/aws-sdk-go/service/ecr/BUILD
	deleted:    vendor/github.com/aws/aws-sdk-go/service/elb/BUILD
	deleted:    vendor/github.com/aws/aws-sdk-go/service/route53/BUILD
	deleted:    vendor/github.com/aws/aws-sdk-go/service/sts/BUILD
	deleted:    vendor/github.com/beorn7/perks/quantile/BUILD
	deleted:    vendor/github.com/blang/semver/BUILD
	deleted:    vendor/github.com/boltdb/bolt/BUILD
	deleted:    vendor/github.com/chai2010/gettext-go/gettext/BUILD
	deleted:    vendor/github.com/chai2010/gettext-go/gettext/mo/BUILD
	deleted:    vendor/github.com/chai2010/gettext-go/gettext/plural/BUILD
	deleted:    vendor/github.com/chai2010/gettext-go/gettext/po/BUILD
	deleted:    vendor/github.com/cloudflare/cfssl/auth/BUILD
	deleted:    vendor/github.com/cloudflare/cfssl/certdb/BUILD
	deleted:    vendor/github.com/cloudflare/cfssl/config/BUILD
	deleted:    vendor/github.com/cloudflare/cfssl/crypto/pkcs7/BUILD
	deleted:    vendor/github.com/cloudflare/cfssl/csr/BUILD
	deleted:    vendor/github.com/cloudflare/cfssl/errors/BUILD
	deleted:    vendor/github.com/cloudflare/cfssl/helpers/BUILD
	deleted:    vendor/github.com/cloudflare/cfssl/helpers/derhelpers/BUILD
	deleted:    vendor/github.com/cloudflare/cfssl/info/BUILD
	deleted:    vendor/github.com/cloudflare/cfssl/log/BUILD
	deleted:    vendor/github.com/cloudflare/cfssl/ocsp/config/BUILD
	deleted:    vendor/github.com/cloudflare/cfssl/signer/BUILD
	deleted:    vendor/github.com/cloudflare/cfssl/signer/local/BUILD
	deleted:    vendor/github.com/clusterhq/flocker-go/BUILD
	deleted:    vendor/github.com/codedellemc/goscaleio/BUILD
	deleted:    vendor/github.com/codedellemc/goscaleio/types/v1/BUILD
	deleted:    vendor/github.com/codegangsta/negroni/BUILD
	deleted:    vendor/github.com/containernetworking/cni/libcni/BUILD
	deleted:    vendor/github.com/containernetworking/cni/pkg/invoke/BUILD
	deleted:    vendor/github.com/containernetworking/cni/pkg/types/020/BUILD
	deleted:    vendor/github.com/containernetworking/cni/pkg/types/BUILD
	deleted:    vendor/github.com/containernetworking/cni/pkg/types/current/BUILD
	deleted:    vendor/github.com/containernetworking/cni/pkg/version/BUILD
	deleted:    vendor/github.com/coreos/etcd/alarm/BUILD
	deleted:    vendor/github.com/coreos/etcd/auth/BUILD
	deleted:    vendor/github.com/coreos/etcd/auth/authpb/BUILD
	deleted:    vendor/github.com/coreos/etcd/client/BUILD
	deleted:    vendor/github.com/coreos/etcd/clientv3/BUILD
	deleted:    vendor/github.com/coreos/etcd/compactor/BUILD
	deleted:    vendor/github.com/coreos/etcd/discovery/BUILD
	deleted:    vendor/github.com/coreos/etcd/error/BUILD
	deleted:    vendor/github.com/coreos/etcd/etcdserver/BUILD
	deleted:    vendor/github.com/coreos/etcd/etcdserver/api/BUILD
	deleted:    vendor/github.com/coreos/etcd/etcdserver/api/v2http/BUILD
	deleted:    vendor/github.com/coreos/etcd/etcdserver/api/v2http/httptypes/BUILD
	deleted:    vendor/github.com/coreos/etcd/etcdserver/api/v3rpc/BUILD
	deleted:    vendor/github.com/coreos/etcd/etcdserver/api/v3rpc/rpctypes/BUILD
	deleted:    vendor/github.com/coreos/etcd/etcdserver/auth/BUILD
	deleted:    vendor/github.com/coreos/etcd/etcdserver/etcdserverpb/BUILD
	deleted:    vendor/github.com/coreos/etcd/etcdserver/membership/BUILD
	deleted:    vendor/github.com/coreos/etcd/etcdserver/stats/BUILD
	deleted:    vendor/github.com/coreos/etcd/integration/BUILD
	deleted:    vendor/github.com/coreos/etcd/lease/BUILD
	deleted:    vendor/github.com/coreos/etcd/lease/leasehttp/BUILD
	deleted:    vendor/github.com/coreos/etcd/lease/leasepb/BUILD
	deleted:    vendor/github.com/coreos/etcd/mvcc/BUILD
	deleted:    vendor/github.com/coreos/etcd/mvcc/backend/BUILD
	deleted:    vendor/github.com/coreos/etcd/mvcc/mvccpb/BUILD
	deleted:    vendor/github.com/coreos/etcd/pkg/adt/BUILD
	deleted:    vendor/github.com/coreos/etcd/pkg/contention/BUILD
	deleted:    vendor/github.com/coreos/etcd/pkg/cpuutil/BUILD
	deleted:    vendor/github.com/coreos/etcd/pkg/crc/BUILD
	deleted:    vendor/github.com/coreos/etcd/pkg/fileutil/BUILD
	deleted:    vendor/github.com/coreos/etcd/pkg/httputil/BUILD
	deleted:    vendor/github.com/coreos/etcd/pkg/idutil/BUILD
	deleted:    vendor/github.com/coreos/etcd/pkg/ioutil/BUILD
	deleted:    vendor/github.com/coreos/etcd/pkg/logutil/BUILD
	deleted:    vendor/github.com/coreos/etcd/pkg/monotime/BUILD
	deleted:    vendor/github.com/coreos/etcd/pkg/netutil/BUILD
	deleted:    vendor/github.com/coreos/etcd/pkg/pathutil/BUILD
	deleted:    vendor/github.com/coreos/etcd/pkg/pbutil/BUILD
	deleted:    vendor/github.com/coreos/etcd/pkg/runtime/BUILD
	deleted:    vendor/github.com/coreos/etcd/pkg/schedule/BUILD
	deleted:    vendor/github.com/coreos/etcd/pkg/testutil/BUILD
	deleted:    vendor/github.com/coreos/etcd/pkg/tlsutil/BUILD
	deleted:    vendor/github.com/coreos/etcd/pkg/transport/BUILD
	deleted:    vendor/github.com/coreos/etcd/pkg/types/BUILD
	deleted:    vendor/github.com/coreos/etcd/pkg/wait/BUILD
	deleted:    vendor/github.com/coreos/etcd/proxy/grpcproxy/BUILD
	deleted:    vendor/github.com/coreos/etcd/proxy/grpcproxy/cache/BUILD
	deleted:    vendor/github.com/coreos/etcd/raft/BUILD
	deleted:    vendor/github.com/coreos/etcd/raft/raftpb/BUILD
	deleted:    vendor/github.com/coreos/etcd/rafthttp/BUILD
	deleted:    vendor/github.com/coreos/etcd/snap/BUILD
	deleted:    vendor/github.com/coreos/etcd/snap/snappb/BUILD
	deleted:    vendor/github.com/coreos/etcd/store/BUILD
	deleted:    vendor/github.com/coreos/etcd/version/BUILD
	deleted:    vendor/github.com/coreos/etcd/wal/BUILD
	deleted:    vendor/github.com/coreos/etcd/wal/walpb/BUILD
	deleted:    vendor/github.com/coreos/go-oidc/http/BUILD
	deleted:    vendor/github.com/coreos/go-oidc/jose/BUILD
	deleted:    vendor/github.com/coreos/go-oidc/key/BUILD
	deleted:    vendor/github.com/coreos/go-oidc/oauth2/BUILD
	deleted:    vendor/github.com/coreos/go-oidc/oidc/BUILD
	deleted:    vendor/github.com/coreos/go-semver/semver/BUILD
	deleted:    vendor/github.com/coreos/go-systemd/daemon/BUILD
	deleted:    vendor/github.com/coreos/go-systemd/dbus/BUILD
	deleted:    vendor/github.com/coreos/go-systemd/journal/BUILD
	deleted:    vendor/github.com/coreos/go-systemd/unit/BUILD
	deleted:    vendor/github.com/coreos/go-systemd/util/BUILD
	deleted:    vendor/github.com/coreos/pkg/capnslog/BUILD
	deleted:    vendor/github.com/coreos/pkg/dlopen/BUILD
	deleted:    vendor/github.com/coreos/pkg/health/BUILD
	deleted:    vendor/github.com/coreos/pkg/httputil/BUILD
	deleted:    vendor/github.com/coreos/pkg/timeutil/BUILD
	deleted:    vendor/github.com/coreos/rkt/api/v1alpha/BUILD
	deleted:    vendor/github.com/cpuguy83/go-md2man/md2man/BUILD
	deleted:    vendor/github.com/davecgh/go-spew/spew/BUILD
	deleted:    vendor/github.com/daviddengcn/go-colortext/BUILD
	deleted:    vendor/github.com/dgrijalva/jwt-go/BUILD
	deleted:    vendor/github.com/docker/distribution/digest/BUILD
	deleted:    vendor/github.com/docker/distribution/reference/BUILD
	deleted:    vendor/github.com/docker/docker/pkg/jsonlog/BUILD
	deleted:    vendor/github.com/docker/docker/pkg/jsonmessage/BUILD
	deleted:    vendor/github.com/docker/docker/pkg/longpath/BUILD
	deleted:    vendor/github.com/docker/docker/pkg/mount/BUILD
	deleted:    vendor/github.com/docker/docker/pkg/stdcopy/BUILD
	deleted:    vendor/github.com/docker/docker/pkg/symlink/BUILD
	deleted:    vendor/github.com/docker/docker/pkg/system/BUILD
	deleted:    vendor/github.com/docker/docker/pkg/term/BUILD
	deleted:    vendor/github.com/docker/docker/pkg/term/windows/BUILD
	deleted:    vendor/github.com/docker/engine-api/client/BUILD
	deleted:    vendor/github.com/docker/engine-api/client/transport/BUILD
	deleted:    vendor/github.com/docker/engine-api/client/transport/cancellable/BUILD
	deleted:    vendor/github.com/docker/engine-api/types/BUILD
	deleted:    vendor/github.com/docker/engine-api/types/blkiodev/BUILD
	deleted:    vendor/github.com/docker/engine-api/types/container/BUILD
	deleted:    vendor/github.com/docker/engine-api/types/filters/BUILD
	deleted:    vendor/github.com/docker/engine-api/types/network/BUILD
	deleted:    vendor/github.com/docker/engine-api/types/reference/BUILD
	deleted:    vendor/github.com/docker/engine-api/types/registry/BUILD
	deleted:    vendor/github.com/docker/engine-api/types/strslice/BUILD
	deleted:    vendor/github.com/docker/engine-api/types/time/BUILD
	deleted:    vendor/github.com/docker/engine-api/types/versions/BUILD
	deleted:    vendor/github.com/docker/go-connections/nat/BUILD
	deleted:    vendor/github.com/docker/go-connections/sockets/BUILD
	deleted:    vendor/github.com/docker/go-connections/tlsconfig/BUILD
	deleted:    vendor/github.com/docker/go-units/BUILD
	deleted:    vendor/github.com/docker/spdystream/BUILD
	deleted:    vendor/github.com/docker/spdystream/spdy/BUILD
	deleted:    vendor/github.com/elazarl/go-bindata-assetfs/BUILD
	deleted:    vendor/github.com/elazarl/goproxy/BUILD
	deleted:    vendor/github.com/emicklei/go-restful-swagger12/BUILD
	deleted:    vendor/github.com/emicklei/go-restful/BUILD
	deleted:    vendor/github.com/emicklei/go-restful/log/BUILD
	deleted:    vendor/github.com/evanphx/json-patch/BUILD
	deleted:    vendor/github.com/exponent-io/jsonpath/BUILD
	deleted:    vendor/github.com/fatih/camelcase/BUILD
	deleted:    vendor/github.com/fsnotify/fsnotify/BUILD
	deleted:    vendor/github.com/garyburd/redigo/internal/BUILD
	deleted:    vendor/github.com/garyburd/redigo/redis/BUILD
	deleted:    vendor/github.com/ghodss/yaml/BUILD
	deleted:    vendor/github.com/go-ini/ini/BUILD
	deleted:    vendor/github.com/go-openapi/analysis/BUILD
	deleted:    vendor/github.com/go-openapi/errors/BUILD
	deleted:    vendor/github.com/go-openapi/jsonpointer/BUILD
	deleted:    vendor/github.com/go-openapi/jsonreference/BUILD
	deleted:    vendor/github.com/go-openapi/loads/BUILD
	deleted:    vendor/github.com/go-openapi/runtime/BUILD
	deleted:    vendor/github.com/go-openapi/spec/BUILD
	deleted:    vendor/github.com/go-openapi/strfmt/BUILD
	deleted:    vendor/github.com/go-openapi/swag/BUILD
	deleted:    vendor/github.com/go-openapi/validate/BUILD
	deleted:    vendor/github.com/godbus/dbus/BUILD
	deleted:    vendor/github.com/gogo/protobuf/gogoproto/BUILD
	deleted:    vendor/github.com/gogo/protobuf/plugin/compare/BUILD
	deleted:    vendor/github.com/gogo/protobuf/plugin/defaultcheck/BUILD
	deleted:    vendor/github.com/gogo/protobuf/plugin/description/BUILD
	deleted:    vendor/github.com/gogo/protobuf/plugin/embedcheck/BUILD
	deleted:    vendor/github.com/gogo/protobuf/plugin/enumstringer/BUILD
	deleted:    vendor/github.com/gogo/protobuf/plugin/equal/BUILD
	deleted:    vendor/github.com/gogo/protobuf/plugin/face/BUILD
	deleted:    vendor/github.com/gogo/protobuf/plugin/gostring/BUILD
	deleted:    vendor/github.com/gogo/protobuf/plugin/marshalto/BUILD
	deleted:    vendor/github.com/gogo/protobuf/plugin/oneofcheck/BUILD
	deleted:    vendor/github.com/gogo/protobuf/plugin/populate/BUILD
	deleted:    vendor/github.com/gogo/protobuf/plugin/size/BUILD
	deleted:    vendor/github.com/gogo/protobuf/plugin/stringer/BUILD
	deleted:    vendor/github.com/gogo/protobuf/plugin/testgen/BUILD
	deleted:    vendor/github.com/gogo/protobuf/plugin/union/BUILD
	deleted:    vendor/github.com/gogo/protobuf/plugin/unmarshal/BUILD
	deleted:    vendor/github.com/gogo/protobuf/proto/BUILD
	deleted:    vendor/github.com/gogo/protobuf/protoc-gen-gogo/descriptor/BUILD
	deleted:    vendor/github.com/gogo/protobuf/protoc-gen-gogo/generator/BUILD
	deleted:    vendor/github.com/gogo/protobuf/protoc-gen-gogo/grpc/BUILD
	deleted:    vendor/github.com/gogo/protobuf/protoc-gen-gogo/plugin/BUILD
	deleted:    vendor/github.com/gogo/protobuf/sortkeys/BUILD
	deleted:    vendor/github.com/gogo/protobuf/vanity/BUILD
	deleted:    vendor/github.com/gogo/protobuf/vanity/command/BUILD
	deleted:    vendor/github.com/golang/glog/BUILD
	deleted:    vendor/github.com/golang/groupcache/lru/BUILD
	deleted:    vendor/github.com/golang/mock/gomock/BUILD
	deleted:    vendor/github.com/golang/protobuf/jsonpb/BUILD
	deleted:    vendor/github.com/golang/protobuf/proto/BUILD
	deleted:    vendor/github.com/golang/protobuf/ptypes/BUILD
	deleted:    vendor/github.com/golang/protobuf/ptypes/any/BUILD
	deleted:    vendor/github.com/golang/protobuf/ptypes/duration/BUILD
	deleted:    vendor/github.com/golang/protobuf/ptypes/timestamp/BUILD
	deleted:    vendor/github.com/google/btree/BUILD
	deleted:    vendor/github.com/google/cadvisor/api/BUILD
	deleted:    vendor/github.com/google/cadvisor/cache/memory/BUILD
	deleted:    vendor/github.com/google/cadvisor/client/v2/BUILD
	deleted:    vendor/github.com/google/cadvisor/collector/BUILD
	deleted:    vendor/github.com/google/cadvisor/container/BUILD
	deleted:    vendor/github.com/google/cadvisor/container/common/BUILD
	deleted:    vendor/github.com/google/cadvisor/container/docker/BUILD
	deleted:    vendor/github.com/google/cadvisor/container/libcontainer/BUILD
	deleted:    vendor/github.com/google/cadvisor/container/raw/BUILD
	deleted:    vendor/github.com/google/cadvisor/container/rkt/BUILD
	deleted:    vendor/github.com/google/cadvisor/container/systemd/BUILD
	deleted:    vendor/github.com/google/cadvisor/devicemapper/BUILD
	deleted:    vendor/github.com/google/cadvisor/events/BUILD
	deleted:    vendor/github.com/google/cadvisor/fs/BUILD
	deleted:    vendor/github.com/google/cadvisor/healthz/BUILD
	deleted:    vendor/github.com/google/cadvisor/http/BUILD
	deleted:    vendor/github.com/google/cadvisor/http/mux/BUILD
	deleted:    vendor/github.com/google/cadvisor/info/v1/BUILD
	deleted:    vendor/github.com/google/cadvisor/info/v2/BUILD
	deleted:    vendor/github.com/google/cadvisor/machine/BUILD
	deleted:    vendor/github.com/google/cadvisor/manager/BUILD
	deleted:    vendor/github.com/google/cadvisor/manager/watcher/BUILD
	deleted:    vendor/github.com/google/cadvisor/manager/watcher/raw/BUILD
	deleted:    vendor/github.com/google/cadvisor/manager/watcher/rkt/BUILD
	deleted:    vendor/github.com/google/cadvisor/metrics/BUILD
	deleted:    vendor/github.com/google/cadvisor/pages/BUILD
	deleted:    vendor/github.com/google/cadvisor/pages/static/BUILD
	deleted:    vendor/github.com/google/cadvisor/storage/BUILD
	deleted:    vendor/github.com/google/cadvisor/summary/BUILD
	deleted:    vendor/github.com/google/cadvisor/utils/BUILD
	deleted:    vendor/github.com/google/cadvisor/utils/cloudinfo/BUILD
	deleted:    vendor/github.com/google/cadvisor/utils/cpuload/BUILD
	deleted:    vendor/github.com/google/cadvisor/utils/cpuload/netlink/BUILD
	deleted:    vendor/github.com/google/cadvisor/utils/docker/BUILD
	deleted:    vendor/github.com/google/cadvisor/utils/oomparser/BUILD
	deleted:    vendor/github.com/google/cadvisor/utils/sysfs/BUILD
	deleted:    vendor/github.com/google/cadvisor/utils/sysinfo/BUILD
	deleted:    vendor/github.com/google/cadvisor/utils/tail/BUILD
	deleted:    vendor/github.com/google/cadvisor/validate/BUILD
	deleted:    vendor/github.com/google/cadvisor/version/BUILD
	deleted:    vendor/github.com/google/cadvisor/zfs/BUILD
	deleted:    vendor/github.com/google/certificate-transparency/go/BUILD
	deleted:    vendor/github.com/google/certificate-transparency/go/asn1/BUILD
	deleted:    vendor/github.com/google/certificate-transparency/go/client/BUILD
	deleted:    vendor/github.com/google/certificate-transparency/go/x509/BUILD
	deleted:    vendor/github.com/google/certificate-transparency/go/x509/pkix/BUILD
	deleted:    vendor/github.com/google/gofuzz/BUILD
	deleted:    vendor/github.com/googleapis/gnostic/OpenAPIv2/BUILD
	deleted:    vendor/github.com/googleapis/gnostic/compiler/BUILD
	deleted:    vendor/github.com/googleapis/gnostic/extensions/BUILD
	deleted:    vendor/github.com/gophercloud/gophercloud/BUILD
	deleted:    vendor/github.com/gophercloud/gophercloud/openstack/BUILD
	deleted:    vendor/github.com/gophercloud/gophercloud/openstack/blockstorage/v1/apiversions/BUILD
	deleted:    vendor/github.com/gophercloud/gophercloud/openstack/blockstorage/v1/volumes/BUILD
	deleted:    vendor/github.com/gophercloud/gophercloud/openstack/blockstorage/v2/volumes/BUILD
	deleted:    vendor/github.com/gophercloud/gophercloud/openstack/common/extensions/BUILD
	deleted:    vendor/github.com/gophercloud/gophercloud/openstack/compute/v2/extensions/volumeattach/BUILD
	deleted:    vendor/github.com/gophercloud/gophercloud/openstack/compute/v2/flavors/BUILD
	deleted:    vendor/github.com/gophercloud/gophercloud/openstack/compute/v2/images/BUILD
	deleted:    vendor/github.com/gophercloud/gophercloud/openstack/compute/v2/servers/BUILD
	deleted:    vendor/github.com/gophercloud/gophercloud/openstack/identity/v2/tenants/BUILD
	deleted:    vendor/github.com/gophercloud/gophercloud/openstack/identity/v2/tokens/BUILD
	deleted:    vendor/github.com/gophercloud/gophercloud/openstack/identity/v3/extensions/trusts/BUILD
	deleted:    vendor/github.com/gophercloud/gophercloud/openstack/identity/v3/tokens/BUILD
	deleted:    vendor/github.com/gophercloud/gophercloud/openstack/networking/v2/extensions/BUILD
	deleted:    vendor/github.com/gophercloud/gophercloud/openstack/networking/v2/extensions/layer3/floatingips/BUILD
	deleted:    vendor/github.com/gophercloud/gophercloud/openstack/networking/v2/extensions/layer3/routers/BUILD
	deleted:    vendor/github.com/gophercloud/gophercloud/openstack/networking/v2/extensions/lbaas/members/BUILD
	deleted:    vendor/github.com/gophercloud/gophercloud/openstack/networking/v2/extensions/lbaas/monitors/BUILD
	deleted:    vendor/github.com/gophercloud/gophercloud/openstack/networking/v2/extensions/lbaas/pools/BUILD
	deleted:    vendor/github.com/gophercloud/gophercloud/openstack/networking/v2/extensions/lbaas/vips/BUILD
	deleted:    vendor/github.com/gophercloud/gophercloud/openstack/networking/v2/extensions/lbaas_v2/listeners/BUILD
	deleted:    vendor/github.com/gophercloud/gophercloud/openstack/networking/v2/extensions/lbaas_v2/loadbalancers/BUILD
	deleted:    vendor/github.com/gophercloud/gophercloud/openstack/networking/v2/extensions/lbaas_v2/monitors/BUILD
	deleted:    vendor/github.com/gophercloud/gophercloud/openstack/networking/v2/extensions/lbaas_v2/pools/BUILD
	deleted:    vendor/github.com/gophercloud/gophercloud/openstack/networking/v2/extensions/security/groups/BUILD
	deleted:    vendor/github.com/gophercloud/gophercloud/openstack/networking/v2/extensions/security/rules/BUILD
	deleted:    vendor/github.com/gophercloud/gophercloud/openstack/networking/v2/ports/BUILD
	deleted:    vendor/github.com/gophercloud/gophercloud/openstack/utils/BUILD
	deleted:    vendor/github.com/gophercloud/gophercloud/pagination/BUILD
	deleted:    vendor/github.com/gorilla/context/BUILD
	deleted:    vendor/github.com/gorilla/mux/BUILD
	deleted:    vendor/github.com/gorilla/websocket/BUILD
	deleted:    vendor/github.com/grpc-ecosystem/go-grpc-prometheus/BUILD
	deleted:    vendor/github.com/grpc-ecosystem/grpc-gateway/runtime/BUILD
	deleted:    vendor/github.com/grpc-ecosystem/grpc-gateway/runtime/internal/BUILD
	deleted:    vendor/github.com/grpc-ecosystem/grpc-gateway/utilities/BUILD
	deleted:    vendor/github.com/hashicorp/golang-lru/BUILD
	deleted:    vendor/github.com/hashicorp/golang-lru/simplelru/BUILD
	deleted:    vendor/github.com/hashicorp/hcl/BUILD
	deleted:    vendor/github.com/hashicorp/hcl/hcl/ast/BUILD
	deleted:    vendor/github.com/hashicorp/hcl/hcl/parser/BUILD
	deleted:    vendor/github.com/hashicorp/hcl/hcl/scanner/BUILD
	deleted:    vendor/github.com/hashicorp/hcl/hcl/strconv/BUILD
	deleted:    vendor/github.com/hashicorp/hcl/hcl/token/BUILD
	deleted:    vendor/github.com/hashicorp/hcl/json/parser/BUILD
	deleted:    vendor/github.com/hashicorp/hcl/json/scanner/BUILD
	deleted:    vendor/github.com/hashicorp/hcl/json/token/BUILD
	deleted:    vendor/github.com/hawkular/hawkular-client-go/metrics/BUILD
	deleted:    vendor/github.com/heketi/heketi/client/api/go-client/BUILD
	deleted:    vendor/github.com/heketi/heketi/pkg/glusterfs/api/BUILD
	deleted:    vendor/github.com/heketi/heketi/pkg/utils/BUILD
	deleted:    vendor/github.com/howeyc/gopass/BUILD
	deleted:    vendor/github.com/imdario/mergo/BUILD
	deleted:    vendor/github.com/inconshreveable/mousetrap/BUILD
	deleted:    vendor/github.com/influxdata/influxdb/client/BUILD
	deleted:    vendor/github.com/influxdata/influxdb/client/v2/BUILD
	deleted:    vendor/github.com/influxdata/influxdb/models/BUILD
	deleted:    vendor/github.com/influxdata/influxdb/pkg/escape/BUILD
	deleted:    vendor/github.com/jmespath/go-jmespath/BUILD
	deleted:    vendor/github.com/jonboulle/clockwork/BUILD
	deleted:    vendor/github.com/jteeuwen/go-bindata/BUILD
	deleted:    vendor/github.com/jteeuwen/go-bindata/go-bindata/BUILD
	deleted:    vendor/github.com/juju/ratelimit/BUILD
	deleted:    vendor/github.com/kardianos/osext/BUILD
	deleted:    vendor/github.com/karlseguin/ccache/BUILD
	deleted:    vendor/github.com/kr/fs/BUILD
	deleted:    vendor/github.com/kr/pty/BUILD
	deleted:    vendor/github.com/libopenstorage/openstorage/api/BUILD
	deleted:    vendor/github.com/libopenstorage/openstorage/api/client/BUILD
	deleted:    vendor/github.com/libopenstorage/openstorage/api/client/volume/BUILD
	deleted:    vendor/github.com/libopenstorage/openstorage/api/spec/BUILD
	deleted:    vendor/github.com/libopenstorage/openstorage/pkg/units/BUILD
	deleted:    vendor/github.com/libopenstorage/openstorage/volume/BUILD
	deleted:    vendor/github.com/lpabon/godbc/BUILD
	deleted:    vendor/github.com/magiconair/properties/BUILD
	deleted:    vendor/github.com/mailru/easyjson/buffer/BUILD
	deleted:    vendor/github.com/mailru/easyjson/jlexer/BUILD
	deleted:    vendor/github.com/mailru/easyjson/jwriter/BUILD
	deleted:    vendor/github.com/matttproud/golang_protobuf_extensions/pbutil/BUILD
	deleted:    vendor/github.com/mesos/mesos-go/detector/BUILD
	deleted:    vendor/github.com/mesos/mesos-go/detector/zoo/BUILD
	deleted:    vendor/github.com/mesos/mesos-go/detector/zoo/client2.go
	deleted:    vendor/github.com/mesos/mesos-go/detector/zoo/detect.go
	deleted:    vendor/github.com/mesos/mesos-go/detector/zoo/doc.go
	deleted:    vendor/github.com/mesos/mesos-go/detector/zoo/plugin.go
	deleted:    vendor/github.com/mesos/mesos-go/detector/zoo/types.go
	deleted:    vendor/github.com/mesos/mesos-go/mesosproto/BUILD
	deleted:    vendor/github.com/mesos/mesos-go/mesosutil/BUILD
	deleted:    vendor/github.com/mesos/mesos-go/upid/BUILD
	deleted:    vendor/github.com/miekg/coredns/middleware/etcd/msg/BUILD
	deleted:    vendor/github.com/miekg/dns/BUILD
	deleted:    vendor/github.com/mistifyio/go-zfs/BUILD
	deleted:    vendor/github.com/mitchellh/go-wordwrap/BUILD
	deleted:    vendor/github.com/mitchellh/mapstructure/BUILD
	deleted:    vendor/github.com/mreiferson/go-httpclient/BUILD
	deleted:    vendor/github.com/mvdan/xurls/BUILD
	deleted:    vendor/github.com/mxk/go-flowrate/flowrate/BUILD
	deleted:    vendor/github.com/onsi/ginkgo/BUILD
	deleted:    vendor/github.com/onsi/ginkgo/config/BUILD
	deleted:    vendor/github.com/onsi/ginkgo/ginkgo/BUILD
	deleted:    vendor/github.com/onsi/ginkgo/ginkgo/convert/BUILD
	deleted:    vendor/github.com/onsi/ginkgo/ginkgo/interrupthandler/BUILD
	deleted:    vendor/github.com/onsi/ginkgo/ginkgo/nodot/BUILD
	deleted:    vendor/github.com/onsi/ginkgo/ginkgo/testrunner/BUILD
	deleted:    vendor/github.com/onsi/ginkgo/ginkgo/testsuite/BUILD
	deleted:    vendor/github.com/onsi/ginkgo/ginkgo/watch/BUILD
	deleted:    vendor/github.com/onsi/ginkgo/internal/codelocation/BUILD
	deleted:    vendor/github.com/onsi/ginkgo/internal/containernode/BUILD
	deleted:    vendor/github.com/onsi/ginkgo/internal/failer/BUILD
	deleted:    vendor/github.com/onsi/ginkgo/internal/leafnodes/BUILD
	deleted:    vendor/github.com/onsi/ginkgo/internal/remote/BUILD
	deleted:    vendor/github.com/onsi/ginkgo/internal/spec/BUILD
	deleted:    vendor/github.com/onsi/ginkgo/internal/spec_iterator/BUILD
	deleted:    vendor/github.com/onsi/ginkgo/internal/specrunner/BUILD
	deleted:    vendor/github.com/onsi/ginkgo/internal/suite/BUILD
	deleted:    vendor/github.com/onsi/ginkgo/internal/testingtproxy/BUILD
	deleted:    vendor/github.com/onsi/ginkgo/internal/writer/BUILD
	deleted:    vendor/github.com/onsi/ginkgo/reporters/BUILD
	deleted:    vendor/github.com/onsi/ginkgo/reporters/stenographer/BUILD
	deleted:    vendor/github.com/onsi/ginkgo/reporters/stenographer/support/go-colorable/BUILD
	deleted:    vendor/github.com/onsi/ginkgo/reporters/stenographer/support/go-isatty/BUILD
	deleted:    vendor/github.com/onsi/ginkgo/types/BUILD
	deleted:    vendor/github.com/onsi/gomega/BUILD
	deleted:    vendor/github.com/onsi/gomega/format/BUILD
	deleted:    vendor/github.com/onsi/gomega/gstruct/BUILD
	deleted:    vendor/github.com/onsi/gomega/gstruct/errors/BUILD
	deleted:    vendor/github.com/onsi/gomega/internal/assertion/BUILD
	deleted:    vendor/github.com/onsi/gomega/internal/asyncassertion/BUILD
	deleted:    vendor/github.com/onsi/gomega/internal/oraclematcher/BUILD
	deleted:    vendor/github.com/onsi/gomega/internal/testingtsupport/BUILD
	deleted:    vendor/github.com/onsi/gomega/matchers/BUILD
	deleted:    vendor/github.com/onsi/gomega/matchers/support/goraph/bipartitegraph/BUILD
	deleted:    vendor/github.com/onsi/gomega/matchers/support/goraph/edge/BUILD
	deleted:    vendor/github.com/onsi/gomega/matchers/support/goraph/node/BUILD
	deleted:    vendor/github.com/onsi/gomega/matchers/support/goraph/util/BUILD
	deleted:    vendor/github.com/onsi/gomega/types/BUILD
	deleted:    vendor/github.com/opencontainers/runc/libcontainer/BUILD
	deleted:    vendor/github.com/opencontainers/runc/libcontainer/apparmor/BUILD
	deleted:    vendor/github.com/opencontainers/runc/libcontainer/cgroups/BUILD
	deleted:    vendor/github.com/opencontainers/runc/libcontainer/cgroups/fs/BUILD
	deleted:    vendor/github.com/opencontainers/runc/libcontainer/cgroups/systemd/BUILD
	deleted:    vendor/github.com/opencontainers/runc/libcontainer/configs/BUILD
	deleted:    vendor/github.com/opencontainers/runc/libcontainer/configs/validate/BUILD
	deleted:    vendor/github.com/opencontainers/runc/libcontainer/criurpc/BUILD
	deleted:    vendor/github.com/opencontainers/runc/libcontainer/keys/BUILD
	deleted:    vendor/github.com/opencontainers/runc/libcontainer/label/BUILD
	deleted:    vendor/github.com/opencontainers/runc/libcontainer/seccomp/BUILD
	deleted:    vendor/github.com/opencontainers/runc/libcontainer/selinux/BUILD
	deleted:    vendor/github.com/opencontainers/runc/libcontainer/stacktrace/BUILD
	deleted:    vendor/github.com/opencontainers/runc/libcontainer/system/BUILD
	deleted:    vendor/github.com/opencontainers/runc/libcontainer/user/BUILD
	deleted:    vendor/github.com/opencontainers/runc/libcontainer/utils/BUILD
	deleted:    vendor/github.com/pborman/uuid/BUILD
	deleted:    vendor/github.com/pelletier/go-buffruneio/BUILD
	deleted:    vendor/github.com/pelletier/go-toml/BUILD
	deleted:    vendor/github.com/pkg/errors/BUILD
	deleted:    vendor/github.com/pkg/sftp/BUILD
	deleted:    vendor/github.com/pmezard/go-difflib/difflib/BUILD
	deleted:    vendor/github.com/prometheus/client_golang/prometheus/BUILD
	deleted:    vendor/github.com/prometheus/client_golang/prometheus/promhttp/BUILD
	deleted:    vendor/github.com/prometheus/client_model/go/BUILD
	deleted:    vendor/github.com/prometheus/common/expfmt/BUILD
	deleted:    vendor/github.com/prometheus/common/internal/bitbucket.org/ww/goautoneg/BUILD
	deleted:    vendor/github.com/prometheus/common/model/BUILD
	deleted:    vendor/github.com/prometheus/procfs/BUILD
	deleted:    vendor/github.com/prometheus/procfs/xfs/BUILD
	deleted:    vendor/github.com/quobyte/api/BUILD
	deleted:    vendor/github.com/rackspace/gophercloud/BUILD
	deleted:    vendor/github.com/rackspace/gophercloud/openstack/BUILD
	deleted:    vendor/github.com/rackspace/gophercloud/openstack/blockstorage/v1/volumes/BUILD
	deleted:    vendor/github.com/rackspace/gophercloud/openstack/compute/v2/extensions/bootfromvolume/BUILD
	deleted:    vendor/github.com/rackspace/gophercloud/openstack/compute/v2/extensions/diskconfig/BUILD
	deleted:    vendor/github.com/rackspace/gophercloud/openstack/compute/v2/extensions/volumeattach/BUILD
	deleted:    vendor/github.com/rackspace/gophercloud/openstack/compute/v2/flavors/BUILD
	deleted:    vendor/github.com/rackspace/gophercloud/openstack/compute/v2/images/BUILD
	deleted:    vendor/github.com/rackspace/gophercloud/openstack/compute/v2/servers/BUILD
	deleted:    vendor/github.com/rackspace/gophercloud/openstack/identity/v2/tenants/BUILD
	deleted:    vendor/github.com/rackspace/gophercloud/openstack/identity/v2/tokens/BUILD
	deleted:    vendor/github.com/rackspace/gophercloud/openstack/identity/v3/tokens/BUILD
	deleted:    vendor/github.com/rackspace/gophercloud/openstack/utils/BUILD
	deleted:    vendor/github.com/rackspace/gophercloud/pagination/BUILD
	deleted:    vendor/github.com/rackspace/gophercloud/rackspace/BUILD
	deleted:    vendor/github.com/rackspace/gophercloud/rackspace/blockstorage/v1/volumes/BUILD
	deleted:    vendor/github.com/rackspace/gophercloud/rackspace/compute/v2/servers/BUILD
	deleted:    vendor/github.com/rackspace/gophercloud/rackspace/compute/v2/volumeattach/BUILD
	deleted:    vendor/github.com/rackspace/gophercloud/rackspace/identity/v2/tokens/BUILD
	deleted:    vendor/github.com/rackspace/gophercloud/testhelper/BUILD
	deleted:    vendor/github.com/rackspace/gophercloud/testhelper/client/BUILD
	deleted:    vendor/github.com/rancher/go-rancher/client/BUILD
	deleted:    vendor/github.com/renstrom/dedent/BUILD
	deleted:    vendor/github.com/robfig/cron/BUILD
	deleted:    vendor/github.com/rubiojr/go-vhd/vhd/BUILD
	deleted:    vendor/github.com/russross/blackfriday/BUILD
	deleted:    vendor/github.com/samuel/go-zookeeper/LICENSE
	deleted:    vendor/github.com/samuel/go-zookeeper/zk/BUILD
	deleted:    vendor/github.com/samuel/go-zookeeper/zk/conn.go
	deleted:    vendor/github.com/samuel/go-zookeeper/zk/constants.go
	deleted:    vendor/github.com/samuel/go-zookeeper/zk/flw.go
	deleted:    vendor/github.com/samuel/go-zookeeper/zk/lock.go
	deleted:    vendor/github.com/samuel/go-zookeeper/zk/server_help.go
	deleted:    vendor/github.com/samuel/go-zookeeper/zk/server_java.go
	deleted:    vendor/github.com/samuel/go-zookeeper/zk/structs.go
	deleted:    vendor/github.com/samuel/go-zookeeper/zk/tracer.go
	deleted:    vendor/github.com/samuel/go-zookeeper/zk/util.go
	deleted:    vendor/github.com/seccomp/libseccomp-golang/BUILD
	deleted:    vendor/github.com/shurcooL/sanitized_anchor_name/BUILD
	deleted:    vendor/github.com/spf13/afero/BUILD
	deleted:    vendor/github.com/spf13/afero/mem/BUILD
	deleted:    vendor/github.com/spf13/afero/sftp/BUILD
	deleted:    vendor/github.com/spf13/cast/BUILD
	deleted:    vendor/github.com/spf13/cobra/BUILD
	deleted:    vendor/github.com/spf13/cobra/doc/BUILD
	deleted:    vendor/github.com/spf13/jwalterweatherman/BUILD
	deleted:    vendor/github.com/spf13/pflag/BUILD
	deleted:    vendor/github.com/spf13/viper/BUILD
	deleted:    vendor/github.com/square/go-jose/BUILD
	deleted:    vendor/github.com/square/go-jose/cipher/BUILD
	deleted:    vendor/github.com/square/go-jose/json/BUILD
	deleted:    vendor/github.com/storageos/go-api/BUILD
	deleted:    vendor/github.com/storageos/go-api/types/BUILD
	deleted:    vendor/github.com/stretchr/objx/BUILD
	deleted:    vendor/github.com/stretchr/testify/assert/BUILD
	deleted:    vendor/github.com/stretchr/testify/mock/BUILD
	deleted:    vendor/github.com/stretchr/testify/require/BUILD
	deleted:    vendor/github.com/syndtr/gocapability/capability/BUILD
	deleted:    vendor/github.com/ugorji/go/codec/BUILD
	deleted:    vendor/github.com/ugorji/go/codec/codecgen/BUILD
	deleted:    vendor/github.com/vishvananda/netlink/BUILD
	deleted:    vendor/github.com/vishvananda/netlink/nl/BUILD
	deleted:    vendor/github.com/vmware/govmomi/BUILD
	deleted:    vendor/github.com/vmware/govmomi/find/BUILD
	deleted:    vendor/github.com/vmware/govmomi/list/BUILD
	deleted:    vendor/github.com/vmware/govmomi/object/BUILD
	deleted:    vendor/github.com/vmware/govmomi/pbm/BUILD
	deleted:    vendor/github.com/vmware/govmomi/pbm/methods/BUILD
	deleted:    vendor/github.com/vmware/govmomi/pbm/types/BUILD
	deleted:    vendor/github.com/vmware/govmomi/property/BUILD
	deleted:    vendor/github.com/vmware/govmomi/session/BUILD
	deleted:    vendor/github.com/vmware/govmomi/task/BUILD
	deleted:    vendor/github.com/vmware/govmomi/vim25/BUILD
	deleted:    vendor/github.com/vmware/govmomi/vim25/debug/BUILD
	deleted:    vendor/github.com/vmware/govmomi/vim25/methods/BUILD
	deleted:    vendor/github.com/vmware/govmomi/vim25/mo/BUILD
	deleted:    vendor/github.com/vmware/govmomi/vim25/progress/BUILD
	deleted:    vendor/github.com/vmware/govmomi/vim25/soap/BUILD
	deleted:    vendor/github.com/vmware/govmomi/vim25/types/BUILD
	deleted:    vendor/github.com/vmware/govmomi/vim25/xml/BUILD
	deleted:    vendor/github.com/vmware/photon-controller-go-sdk/SSPI/BUILD
	deleted:    vendor/github.com/vmware/photon-controller-go-sdk/photon/BUILD
	deleted:    vendor/github.com/vmware/photon-controller-go-sdk/photon/lightwave/BUILD
	deleted:    vendor/github.com/xanzy/go-cloudstack/cloudstack/BUILD
	deleted:    vendor/github.com/xiang90/probing/BUILD
	deleted:    vendor/github.com/xyproto/simpleredis/BUILD
	deleted:    vendor/go.pedge.io/pb/go/google/protobuf/BUILD
	deleted:    vendor/go4.org/errorutil/BUILD
	deleted:    vendor/golang.org/x/crypto/bcrypt/BUILD
	deleted:    vendor/golang.org/x/crypto/blowfish/BUILD
	deleted:    vendor/golang.org/x/crypto/curve25519/BUILD
	deleted:    vendor/golang.org/x/crypto/ed25519/BUILD
	deleted:    vendor/golang.org/x/crypto/ed25519/internal/edwards25519/BUILD
	deleted:    vendor/golang.org/x/crypto/nacl/secretbox/BUILD
	deleted:    vendor/golang.org/x/crypto/pkcs12/BUILD
	deleted:    vendor/golang.org/x/crypto/pkcs12/internal/rc2/BUILD
	deleted:    vendor/golang.org/x/crypto/poly1305/BUILD
	deleted:    vendor/golang.org/x/crypto/salsa20/salsa/BUILD
	deleted:    vendor/golang.org/x/crypto/ssh/BUILD
	deleted:    vendor/golang.org/x/crypto/ssh/terminal/BUILD
	deleted:    vendor/golang.org/x/exp/inotify/BUILD
	deleted:    vendor/golang.org/x/net/context/BUILD
	deleted:    vendor/golang.org/x/net/context/ctxhttp/BUILD
	deleted:    vendor/golang.org/x/net/html/BUILD
	deleted:    vendor/golang.org/x/net/html/atom/BUILD
	deleted:    vendor/golang.org/x/net/http2/BUILD
	deleted:    vendor/golang.org/x/net/http2/hpack/BUILD
	deleted:    vendor/golang.org/x/net/idna/BUILD
	deleted:    vendor/golang.org/x/net/internal/timeseries/BUILD
	deleted:    vendor/golang.org/x/net/lex/httplex/BUILD
	deleted:    vendor/golang.org/x/net/proxy/BUILD
	deleted:    vendor/golang.org/x/net/trace/BUILD
	deleted:    vendor/golang.org/x/net/websocket/BUILD
	deleted:    vendor/golang.org/x/oauth2/BUILD
	deleted:    vendor/golang.org/x/oauth2/google/BUILD
	deleted:    vendor/golang.org/x/oauth2/internal/BUILD
	deleted:    vendor/golang.org/x/oauth2/jws/BUILD
	deleted:    vendor/golang.org/x/oauth2/jwt/BUILD
	deleted:    vendor/golang.org/x/sys/unix/BUILD
	deleted:    vendor/golang.org/x/sys/windows/BUILD
	deleted:    vendor/golang.org/x/text/cases/BUILD
	deleted:    vendor/golang.org/x/text/encoding/BUILD
	deleted:    vendor/golang.org/x/text/encoding/internal/BUILD
	deleted:    vendor/golang.org/x/text/encoding/internal/identifier/BUILD
	deleted:    vendor/golang.org/x/text/encoding/unicode/BUILD
	deleted:    vendor/golang.org/x/text/internal/tag/BUILD
	deleted:    vendor/golang.org/x/text/internal/utf8internal/BUILD
	deleted:    vendor/golang.org/x/text/language/BUILD
	deleted:    vendor/golang.org/x/text/runes/BUILD
	deleted:    vendor/golang.org/x/text/secure/bidirule/BUILD
	deleted:    vendor/golang.org/x/text/secure/precis/BUILD
	deleted:    vendor/golang.org/x/text/transform/BUILD
	deleted:    vendor/golang.org/x/text/unicode/bidi/BUILD
	deleted:    vendor/golang.org/x/text/unicode/norm/BUILD
	deleted:    vendor/golang.org/x/text/width/BUILD
	deleted:    vendor/golang.org/x/time/rate/BUILD
	deleted:    vendor/golang.org/x/tools/container/intsets/BUILD
	deleted:    vendor/google.golang.org/api/cloudkms/v1/BUILD
	deleted:    vendor/google.golang.org/api/cloudmonitoring/v2beta2/BUILD
	deleted:    vendor/google.golang.org/api/compute/v0.beta/BUILD
	deleted:    vendor/google.golang.org/api/compute/v1/BUILD
	deleted:    vendor/google.golang.org/api/container/v1/BUILD
	deleted:    vendor/google.golang.org/api/dns/v1/BUILD
	deleted:    vendor/google.golang.org/api/gensupport/BUILD
	deleted:    vendor/google.golang.org/api/googleapi/BUILD
	deleted:    vendor/google.golang.org/api/googleapi/internal/uritemplates/BUILD
	deleted:    vendor/google.golang.org/api/logging/v2beta1/BUILD
	deleted:    vendor/google.golang.org/api/monitoring/v3/BUILD
	deleted:    vendor/google.golang.org/api/pubsub/v1/BUILD
	deleted:    vendor/google.golang.org/grpc/BUILD
	deleted:    vendor/google.golang.org/grpc/codes/BUILD
	deleted:    vendor/google.golang.org/grpc/credentials/BUILD
	deleted:    vendor/google.golang.org/grpc/grpclog/BUILD
	deleted:    vendor/google.golang.org/grpc/internal/BUILD
	deleted:    vendor/google.golang.org/grpc/metadata/BUILD
	deleted:    vendor/google.golang.org/grpc/naming/BUILD
	deleted:    vendor/google.golang.org/grpc/peer/BUILD
	deleted:    vendor/google.golang.org/grpc/transport/BUILD
	deleted:    vendor/gopkg.in/gcfg.v1/BUILD
	deleted:    vendor/gopkg.in/gcfg.v1/scanner/BUILD
	deleted:    vendor/gopkg.in/gcfg.v1/token/BUILD
	deleted:    vendor/gopkg.in/gcfg.v1/types/BUILD
	deleted:    vendor/gopkg.in/inf.v0/BUILD
	deleted:    vendor/gopkg.in/natefinch/lumberjack.v2/BUILD
	deleted:    vendor/gopkg.in/yaml.v2/BUILD
	deleted:    vendor/k8s.io/gengo/args/BUILD
	deleted:    vendor/k8s.io/gengo/examples/deepcopy-gen/generators/BUILD
	deleted:    vendor/k8s.io/gengo/examples/defaulter-gen/generators/BUILD
	deleted:    vendor/k8s.io/gengo/examples/import-boss/generators/BUILD
	deleted:    vendor/k8s.io/gengo/examples/set-gen/generators/BUILD
	deleted:    vendor/k8s.io/gengo/examples/set-gen/sets/BUILD
	deleted:    vendor/k8s.io/gengo/generator/BUILD
	deleted:    vendor/k8s.io/gengo/namer/BUILD
	deleted:    vendor/k8s.io/gengo/parser/BUILD
	deleted:    vendor/k8s.io/gengo/types/BUILD
	deleted:    vendor/k8s.io/heapster/metrics/api/v1/types/BUILD
	deleted:    vendor/vbom.ml/util/sortorder/BUILD
```

k82cn commented:
Run godep-restore.sh ahead.

gyliu513 commented:
@k82cn did not quite catch what do you mean, can you explain more?

So I need first run `godep-restore.sh` then `godep-save.sh`? 

But when I run `godep-restore.sh`, it failed.

```
root@pc005:~/go/src/k8s.io/kubernetes# ./hack/godep-restore.sh
Starting to download all kubernetes godeps. This takes a while
godep: error downloading dep (vbom.ml/util/sortorder): unrecognized import path "vbom.ml/util/sortorder"
godep: Error downloading some deps. Aborting restore and check.
!!! Error in ./hack/godep-restore.sh:28
  Error in ./hack/godep-restore.sh:28. 'GOPATH=${GOPATH}:${KUBE_ROOT}/staging godep restore "$@"' exited with status 1
Call stack:
  1: ./hack/godep-restore.sh:28 main(...)
Exiting with status 1
```

k82cn commented:
Because of GFW ?

sttts commented:
The deleted files are all bazel BUILD files. Do a `hack/update-bazel.sh` after `godep-save.sh`.

gyliu513 commented:
Thanks @sttts and @k82cn 

I think what @sttts proposed does make sense as my VM was located in US and there is no GFW. After I finished both `./hack/godep-save.sh` and `hack/update-godep-licenses.sh`, then when I run `hack/update-bazel.sh`, it failed again as following, complaining cannot find `vendor/github.com/docker/docker/project/CONTRIBUTORS.md` as this file was already deleted when I run `./hack/godep-save.sh`.

```
root@pc005:~/go/src/k8s.io/kubernetes# ./hack/update-bazel.sh
F0710 03:14:50.252162   22870 gazel.go:54] err walking vendor: stat vendor/github.com/docker/docker/project/CONTRIBUTORS.md: no such file or directory
goroutine 1 [running]:
gopkg.in/mikedanese/gazel.v17/vendor/github.com/golang/glog.stacks(0xc42052ea00, 0xc4202893b0, 0x97, 0xeb)
	/root/go/src/gopkg.in/mikedanese/gazel.v17/vendor/github.com/golang/glog/glog.go:766 +0xa7
gopkg.in/mikedanese/gazel.v17/vendor/github.com/golang/glog.(*loggingT).output(0x6c05a0, 0xc400000003, 0xc4202e3600, 0x6a2614, 0x8, 0x36, 0x0)
	/root/go/src/gopkg.in/mikedanese/gazel.v17/vendor/github.com/golang/glog/glog.go:717 +0x348
gopkg.in/mikedanese/gazel.v17/vendor/github.com/golang/glog.(*loggingT).printf(0x6c05a0, 0xc400000003, 0x5dd1ff, 0x16, 0xc4200f1f38, 0x1, 0x1)
	/root/go/src/gopkg.in/mikedanese/gazel.v17/vendor/github.com/golang/glog/glog.go:655 +0x14f
gopkg.in/mikedanese/gazel.v17/vendor/github.com/golang/glog.Fatalf(0x5dd1ff, 0x16, 0xc4200f1f38, 0x1, 0x1)
	/root/go/src/gopkg.in/mikedanese/gazel.v17/vendor/github.com/golang/glog/glog.go:1145 +0x67
main.main()
	/root/go/src/gopkg.in/mikedanese/gazel.v17/gazel/gazel.go:54 +0x2cd
!!! Error in ./hack/update-bazel.sh:40
  Error in ./hack/update-bazel.sh:40. '"${gazel}" -root="$(kube::realpath ${KUBE_ROOT})"' exited with status 255
Call stack:
  1: ./hack/update-bazel.sh:40 main(...)
Exiting with status 1
```

The steps I did was as following:

Step 1: Remove Godeps/ and vendor/, then run `./hack/godep-save.sh`
```
root@pc005:~/go/src/k8s.io/kubernetes# ./hack/godep-save.sh
Don't forget to run hack/update-godep-licenses.sh if you added or removed a dependency!
```

Step 2: Run `hack/update-godep-licenses.sh`
```
root@pc005:~/go/src/k8s.io/kubernetes# hack/update-godep-licenses.sh
```

Step 3: Run `hack/update-bazel.sh` failed as above.

@sttts any comments for this? Thanks!

sttts commented:
Known issue. Solution: delete vendor/github.com/docker/docker/project/

sttts commented:
@gyliu513 can you test https://github.com/kubernetes/kubernetes/pull/48685?

gyliu513 committed:
Removed old mesos deps.

gyliu513 commented:
/retest

luxas commented:
@gyliu513 I think that worked :tada:
The test failures are unrelated AFAICS

sttts commented:
Yes, tests are broken https://github.com/kubernetes/kubernetes/pull/48683 and federation times out or looses connection.

mikedanese commented:
Love the diff :)

sttts commented:
Love the -68k lines of code.

k82cn commented:
+1 :).

gyliu513 commented:
/retest

luxas commented:
/retest

gyliu513 commented:
/retest

gyliu513 commented:
ping @luxas and @mikedanese for lgtm and approve

luxas commented:
/lgtm

luxas commented:
As everyone here is happy with this, I approved as well
